phantomjs/bridge.js

```diff
@@ -9,7 +9,7 @@
 /*global QUnit:true, alert:true*/
 (function (factory) {
   if (typeof define === 'function' && define.amd) {
-    require(['qunit'], factory);
+    define('phantom-bridge', ['qunit'], factory);
```

Comment:
Totally understood. Thanks for the explanation :)

Could you add a test for it so we can link to it, after, in docs (section `Loading QUnit with AMD`) as a usage example?